prismic-model/src/parts/bodies/index.ts

```diff
@@ -0,0 +1,7 @@
+// If a new body gets added, don't forget to ensure it's added in sliceAnalysis.ts
```

Comment:
Not for this PR, but I'm wondering if we need three different 'bodies' or whether they could just be unified into one - I can't remember what the differences are